db/migrations/sqlite3/0007_user_dates.up.sql

```diff
@@ -0,0 +1,28 @@
+PRAGMA foreign_keys=off;
```

Comment:
This likely does nothing since we're already in a transaction

Reply:
Also, the current connection string does not turn foreign keys on anyway. The "fix" may be to modify the connection string and change this to a defer_foreign_keys instead, so the foreign keys are only checked up on commit of this transaction.